src/shared/components/search-result/search-result.component.html

```diff
@@ -1,6 +1,6 @@
 <table *ngIf="hasResults() || hasDrafts()">
   <caption>
-    <h2 class="heading-h2">{{ caseState ? 'Case List' : 'Search result' }}</h2>
+    <h2 class="heading-h2">{{ caseState ? 'Your cases' : 'Search result' }}</h2>
```

Comment:
Being picky, shouldn't the false state say 'Search results' to maintain the plural nature? For the sake of consistency.

Reply:
Thank's Paul I wasn't asked to change that however I'll mention your comment on the ticket and include the necessary people to bring it to their attention.